### ISSUE TITLE
Owncloud Client not deleting files - connection timed out

### ISSUE THREAD

**Issue by fredericmohr** [owncloud/client#2748]
Same problem as https://github.com/owncloud/client/issues/2647

# Actual behaviour

Error message and frozen sync client with message "Could not read status line: connection timed out. General: Could not read status line. Connection Timed out."

# Steps to reproduce

I copied multiple folder to a new location, and deleted the originals afterwards. In two different Sync folders "Work" and "Documents", one had to delete 20 files, the other around 2000. Sync of the new files went without problems, but deleting files always takes forever if it can finish it at all. Mostly ends in "connection timed out", with oc client alerting that sync broke.

# Server configuration
Operating system: Ubuntu 14.04.1 LTS
Web server: 2.4.7-1ubuntu4.1
Database: MariaDB 5.5.40-0ubuntu0.14.04.1
PHP version: Installed: 5.5.9+dfsg-1ubuntu4.5
ownCloud version: ownCloud 7.0.4 (stable)
Storage backend: Just /var/www/owncloud/data folder on main drive.

# Client configuration
Client version: 1.7.1
Operating system: Kubuntu 14.04.1 LTS
OS language: English
Installation path of client: /usr/bin/owncloud

# More info:

I'm connecting via SSH tunnel to my server (proxying to owncloud.lan via myserver.com). Don't think that's the problem here, but wanted to add all the info.
![oc_del04](https://cloud.githubusercontent.com/assets/5955966/5900431/61f46ba8-a569-11e4-9f19-19ce1f39af63.png)

    # SSH Tunnel for owncloud
    ssh -L 9090:localhost:443 owncloud.lan -N
    
    # My SSH Config
    Host *.lan
        ProxyCommand ssh user@myserver.com nc %h %p

Here are some screenshots I made...
**Owncloud Error**
![oc_del01](https://cloud.githubusercontent.com/assets/5955966/5900433/61fa2386-a569-11e4-9f2f-faf94c69345f.png)

**Deleting files manually on server**
![oc_del02](https://cloud.githubusercontent.com/assets/5955966/5900432/61f638fc-a569-11e4-8d3f-fee62354147e.png)

**Owncloud after restarting client**
![oc_03](https://cloud.githubusercontent.com/assets/5955966/5900430/61ea4cd6-a569-11e4-8253-e5966d248d7b.png)

**Comment by guruz** [comment]
Thanks for your report..

@PVince81 @dragotin Is this similar to the server-trashbin issue?

**Comment by dragotin** [comment]
I think so. @fredericmohr can you check how much files you have in the trashbin?

**Comment by fredericmohr** [comment]
How exactly, do you mean KDE trash on client or server side? Is this a owncloud-trash thing?

**Comment by guruz** [comment]
@fredericmohr server trashbin inside owncloud :)

**Comment by fredericmohr** [comment]
![del1](https://cloud.githubusercontent.com/assets/5955966/5917475/33a22ef2-a61d-11e4-9896-ef5786d68c3c.png)

This is what it looks like in owncloud. These are the two folders I deleted. It's the same thing when I scroll all the way up to the top. Looks like it's starting all over after each time out. Would be better if it'd continue where it left of. (Splitting deletion into chunks?)

For now - can I disable it? Or is there something else I can do?

**Comment by guruz** [comment]
@dragotin @PVince81 So is it the "trashbin issue" or not? Can we tell @fredericmohr a server version where this is fixed?

@fredericmohr That's a screenshot of the trash? Can you empty it? When it is empty, does the delete go through then?

**Comment by fredericmohr** [comment]
So I had this problem again, with only one emtpy file. But it seems to have resolved itself. Don't know how.
![del1](https://cloud.githubusercontent.com/assets/5955966/5966093/a70598f8-a801-11e4-9c52-27e1d2a27c5c.png)

I could emtpy the trashbin through the webinterface, although it was loading forever. After hitting F5 it showed an emtpy bin, so it worked.

**Comment by fredericmohr** [comment]
Ok, just as I wrote the above, this happened.
![del1](https://cloud.githubusercontent.com/assets/5955966/5966153/222642b2-a802-11e4-8e81-ae447fcb5b46.png)
Checked the trash bin - still emtpy (emptied it maybe 2 minutes ago...)
pause/resume on client sync
![del1](https://cloud.githubusercontent.com/assets/5955966/5966161/38afe61e-a802-11e4-9fbe-0e2f1dea81c0.png)
Checked trash again - still emtpy
![del1](https://cloud.githubusercontent.com/assets/5955966/5966167/3facd724-a802-11e4-9e15-70d9a01009c7.png)

Hope this helps...

**Comment by fredericmohr** [comment]
And just to test the above, everything seems to work fine know...
![del1](https://cloud.githubusercontent.com/assets/5955966/5966190/797cc482-a802-11e4-8d2e-c295d5274531.png)
created, copied, renamed  and deleted a emtpy file...

**Comment by marcorperez** [comment]
Hello, I had this exact same problem as well... emptied the trash bin and seems to work fine for now.

**Comment by eigengrau** [comment]
I occasionally had this problem when running 7.0 on the server. After upgrading to 8.0, it seemed to have exacerbated. However, I would not get the timeout; instead, the deletion prompt would just stall on the client. Deleting a single file took upwards of a minute. Today, I deleted ~20 files, and the sync operation would not finish on client anymore. Even after killing the client and restarting it, nothing would go through. I now emptied the trash bin on the server (~600 files) and the sync went through right away. I’ll have to see and wait whether the issue comes up again.
